src/server/routes/auth.js

```diff
@@ -46,6 +46,13 @@ router.get('/cb', (req, res, next) => {
 				return next(loginErr);
 			}
 
+			const {Editor} = req.app.locals.orm;
+			const lastLoginDate = new Date().toISOString();
+			(() => {
```

Comment:
I'm not sure how much I like self-invoking functions. I find they are confusing, especially for a beginner reading the code for the first time.

I would suggest to make the callback passed to .logIn asynchronous, and `await`ing the Editor function calls.

In pseudo code, just to make sure I am clear:
```
return req.logIn(user, async (loginErr) => {
[...]
await Editor.where(XYZ).save(XYZ);
[...]
return res.redirect(redirectTo);
```